### PR TITLE
feat: add changelogs and readmes

### PR DESCRIPTION
**Pull request by ltchang2019**
## Motivation

Preparing agents for initial release.

## Solution

- Add changelogs to all packages
- Add readmes to all agents
- Port docs from docs.nomad.xyz to rust repo

## PR Checklist

- [ ] Added Tests
- [x] Updated Documentation
- [x] Updated CHANGELOG.md for the appropriate package

### CODE REVIEW

File: agents/RUNNING-AGENTS.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
+# Running Agents
```
Comment (ltchang2019) [inline agents/RUNNING-AGENTS.md:1]: Ported from https://docs.nomad.xyz/dev/agents/running-agents.html

File: agents/watcher/RUNNING-WATCHER.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+**Reasoning for Funding Amounts**
+
+The highest daily average gas price on Ethereum to-date is ~710 gwei. A watcher `unenrollReplica` transaction is ~120k gas while a `doubleUpdate` transaction is ~200k gas. If we 10x the highest daily average gas price, we get 7100 gwei. This means that calling `unenrollReplica` will cost 0.852 ETH and calling `doubleUpdate` will cost 1.42 ETH.
+
+unenrollReplica: (710 x 10 x 120,000) / 1e9 = 0.852 ETH
+
+doubleUpdate: (710 x 10 x 200,000) / 1e9 = 1.42 ETH
+
+A minimum of 3 ETH worth of funds per watcher transaction signer is recommended. For networks outside of Ethereum, the funding amount is inflated due to the fact that the dollar cost of funds is much cheaper on other chains.
```
Comment (ltchang2019) [inline agents/watcher/RUNNING-WATCHER.md:75]: Review here please

File: configuration/src/gas/GAS-HARDCODES.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
+# Agent Gas Limit Values
```
Comment (ltchang2019) [inline configuration/src/gas/GAS-HARDCODES.md:1]: Ported from https://docs.nomad.xyz/dev/agents/gas-values.html

Comment (prestwich) [reply configuration/src/gas/GAS-HARDCODES.md:1]: this should not be in `src/`

**Review by prestwich — CHANGES_REQUESTED** [review]

File: agents/AGENT-OPERATIONS.md [hunk 4]
```diff
@@ -0,0 +1,360 @@
+# Agent Operations
```
[the remark this answers is not in the document]
Comment (prestwich) [reply agents/AGENT-OPERATIONS.md:1]: what's the purpose of reproducing here? Some version will fall out of date no? We shouldn't have to update docs in 2 places

[the remark this answers is not in the document]
Comment (ltchang2019) [reply agents/AGENT-OPERATIONS.md:1]: Will delete all these from docs site. Part of docs overhaul @odyslam has been planning

[the remark this answers is not in the document]
Comment (ltchang2019) [reply agents/AGENT-OPERATIONS.md:1]: We want docs closer to code is TL;DR

**Review by prestwich — APPROVED** [review]

**Pull request merged**